src/CUBE.sol

```diff
@@ -266,7 +288,7 @@ contract CUBE is
 
     /// @notice Returns the version of the CUBE smart contract
     function cubeVersion() external pure returns (string memory) {
-        return "4";
+        return "4.1";
```

Comment:
keep an eye on the cube versions in the admin dashboard so this is picked up correctly. Also ask yourself if any app code needs to change when a contract is "4.1" and not "4"